settings/installer/packstack/packstack.yml

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
We have firewall arg in packstack.spec. To firewall rules will be added as installer.firewall.rules options.